week3/cardgame/src/App.jsx

```diff
@@ -0,0 +1,47 @@
+import { Global, ThemeProvider } from '@emotion/react';
+import { useState, useEffect } from "react";
+import globalStyle from "./styles/globalStyle";
+import theme from "./styles/theme";
+import styled from "@emotion/styled";
+import Header from "./Header"
+import CardTable from "./CardTable";
+import Modal from "./Modal";
+
+
+
+function App() {
+  const [score, setScore] = useState(0);
+  const [showModal, setShowModal] = useState(false);
+
+  useEffect(() => {
+    if (score === 5) {
+      setShowModal(true);
+    }
+  }, [score]);
+
+  const handleCloseModal = () => {
+    setShowModal(false);
+    window.location.reload()
+  };
+  
+  return (
+    <>
+    <Global styles={globalStyle} />
+    <ThemeProvider theme = {theme}>
+      <ContentWrapper>
+        <Header score = {score}/>
+        <CardTable setScore = {setScore} />
+      </ContentWrapper>
+        {showModal && <Modal onClose={handleCloseModal} />}
+    </ThemeProvider>
+    </>
+  );
+}
+
+export default App;
+
+const ContentWrapper = styled.div`
+display: flex;
+flex-direction: column;
+align-items: center;
+`
```

Comment:
p5: 커밋에서만 이렇게 보이는지 모르겠는데 들여쓰기하면 좋을 것 같은데 어떨까~?
```suggestion
const ContentWrapper = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
`
```

Reply:
오옹 좋네요~~! 반영하겠습니다 감사해요! 🤩